app/Components/AddForm.tsx

```diff
@@ -0,0 +1,317 @@
+              name="phoneNumber"
+              id="phoneNumber"
+              placeholder="Enter Employee's Phone Number"
+              value={newEmployee.phoneNumber}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="p-2">
+            <label htmlFor="emergencyPhoneNumber">Emergency Number</label>
+            <input
+              className={formStyle}
+              required
+              type="text"
+              name="emergencyPhoneNumber"
+              id="emergencyPhoneNumber"
+              placeholder="+2349859890391"
+              value={newEmployee.emergencyPhoneNumber}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="p-2">
+            <label htmlFor="bankName">Bank Name</label>
+            <input
+              className={formStyle}
+              required
+              type="text"
+              name="bankName"
+              id="bankName"
+              placeholder="Abc bank"
+              value={newEmployee.bankName}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="p-2">
+            <label htmlFor="bankAccountNumber">Bank Account Number</label>
+            <input
+              className={formStyle}
+              required
+              type="number"
+              name="bankAccountNumber"
+              id="bankAccountNumber"
+              placeholder="10000000001"
+              value={newEmployee.bankAccountNumber}
+              onChange={(e) =>
+                setNewEmployee((prev) => ({
+                  ...prev,
+                  bankAccountNumber: Number(e.target.value),
+                }))
+              }
+            />
+          </div>
+          <div className="p-2">
+            <label htmlFor="accountName">Account Name</label>
+            <input
+              className={formStyle}
+              required
+              type="text"
+              name="accountName"
+              id="accountName"
+              placeholder="Joe biden"
+              value={newEmployee.accountName}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="p-2">
+            <label htmlFor="physicalAddress">Physical Address</label>
+            <input
+              className={formStyle}
+              required
+              type="text"
+              name="physicalAddress"
+              id="physicalAddress"
+              placeholder="Your home address"
+              value={newEmployee.physicalAddress}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="p-2">
+            <fieldset>
+              <legend>Gender</legend>
+              <input
+                type="radio"
+                className="mx-2"
+                name="gender"
+                id="gender"
+                value="MALE"
+                checked={newEmployee.gender === "MALE"}
```

Comment:
Kindly create a type called Gender
This can then be
 checked={newEmployee.gender === Gender.MALE}
 
 Do the same for female